src/drivers/intel/cavs/ipc.c

```diff
@@ -23,6 +23,29 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#if CAVS_VERSION > CAVS_VERSION_1_5
```

Comment:
It's just the optics, but `#if CAVS_VERSION >= CAVS_VERSION_1_8` looks more obvious.

Reply:
Good idea. I separated the new stuff from the refactoring work. It will be delivered (changed as suggested) in another PR soon.